src/server/plugins/feed/index.js

```diff
@@ -28,8 +28,22 @@ type Socket = {
   on: (event: string, cb: (d?: any) => void) => void
 }
 
+type FetchOptions = {
+  from: Date,
+  to: Date,
+  published: Boolean,
+  viewed: Boolean
+}
+
 const logger: Logger = debug('server:plugins:feed')
 
+const fetchOptions: FetchOptions = {
+  from: '2016-09-17T17:26:17+00:00',
+  to: '2016-09-20T17:26:17+00:00',
+  published: true,
+  viewed: false
```

Comment:
We probably don't want viewed: false here. I think we should only filter by one thing: published: true.